enterprise/server/content_addressable_storage_server_proxy/content_addressable_storage_server_proxy_test.go

```diff
@@ -259,6 +260,47 @@ func TestFindMissingBlobs(t *testing.T) {
 	expectNoAtimeUpdate(t, clock, requestCount)
 }
 
+func TestFindMissingBlobs_SkipRemote(t *testing.T) {
+	ctx := testContext()
+	remoteConn, _, _ := runRemoteCASS(ctx, testenv.GetTestEnv(t), t)
+	proxyEnv := testenv.GetTestEnv(t)
+	clock := clockwork.NewFakeClock()
+	proxyEnv.SetClock(clock)
```

Comment:
nit: if you're not worried about the atime updater, you should be able to get rid of this clock stuff and the `flags.Set()` on line 270